Examples/Nearest_Neighbour_Distance.ipynb

```diff
@@ -31,7 +31,9 @@
    },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    import pandas as pd
</pre><p>Remove as not used</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/363/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>